tests/integration/prepare/test_prepare_s2a_l1c_safe.py

```diff
@@ -0,0 +1,344 @@
+                        [493180.2237068879, 6681630.058289605],
+                        [493214.79452054796, 6700000.0],
+                        [399960.0, 6700000.0],
+                        [399960.0, 6590200.0],
+                        [470826.97121060337, 6590200.0]
+                    ]],
+                    'type': 'Polygon'
+                }
+            }
+        },
+        'id': 'd6004c21-171f-5a76-8e4f-2de359af506a',
+        'image': {
+            'bands': {
+                'B01': {
+                    'layer': 1,
+                    'path': path_offset('IMG_DATA/T56JMM_20180629T000241_B01.jp2')
+                },
+                'B02': {
+                    'layer': 1,
+                    'path': path_offset('IMG_DATA/T56JMM_20180629T000241_B02.jp2')
+                },
+                'B03': {
+                    'layer': 1,
+                    'path': path_offset('IMG_DATA/T56JMM_20180629T000241_B03.jp2')
+                },
+                'B04': {
+                    'layer': 1,
+                    'path': path_offset('IMG_DATA/T56JMM_20180629T000241_B04.jp2')
+                },
+                'B05': {
+                    'layer': 1,
+                    'path': path_offset('IMG_DATA/T56JMM_20180629T000241_B05.jp2')
+                },
+                'B06': {
+                    'layer': 1,
+                    'path': path_offset('IMG_DATA/T56JMM_20180629T000241_B06.jp2')
+                },
+                'B07': {
+                    'layer': 1,
+                    'path': path_offset('IMG_DATA/T56JMM_20180629T000241_B07.jp2')
+                },
+                'B08': {
+                    'layer': 1,
+                    'path': path_offset('IMG_DATA/T56JMM_20180629T000241_B08.jp2')
+                },
+                'B09': {
+                    'layer': 1,
+                    'path': path_offset('IMG_DATA/T56JMM_20180629T000241_B09.jp2')
+                },
+                'B10': {
+                    'layer': 1,
+                    'path': path_offset('IMG_DATA/T56JMM_20180629T000241_B10.jp2')
+                },
+                'B11': {
+                    'layer': 1,
+                    'path': path_offset('IMG_DATA/T56JMM_20180629T000241_B11.jp2')
+                },
+                'B12': {
+                    'layer': 1,
+                    'path': path_offset('IMG_DATA/T56JMM_20180629T000241_B12.jp2')
+                },
+                'B8A': {
+                    'layer': 1,
+                    'path': path_offset('IMG_DATA/T56JMM_20180629T000241_B8A.jp2')
+                },
+                'PVI': {
+                    'layer': 1,
+                    'path': path_offset('QI_DATA/T56JMM_20180629T000241_PVI.jp2')
+                },
+                'TCI': {
+                    'layer': 1,
+                    'path': path_offset('IMG_DATA/T56JMM_20180629T000241_TCI.jp2')
+                }
+            },
+            'cloud_cover_percentage': '0.0',
```

Comment:
It wasn't obvious until you added a test for it, but these properties were numbers in the previous collection rather than strings.

From the old tests: https://github.com/GeoscienceAustralia/eo-datasets/blob/1062c02160380b7ebbf2076b2c03ddb4f1bb32bf/tests/integration/test_ls8_nbar.py#L540-L543

I'm guessing this affects all of these newly-added prepare scripts. We can address this in a separate issue if you want. Many of the newly added fields in this section probably need the same change.